src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -214,14 +214,18 @@ def _from_directory_slot(self) -> None:
         self._model.set_input_mode(InputMode.FROM_PATH)
 
     def _update_layer_list(self, event: Optional[NapariEvent] = None) -> None:
-        existing_selection = [
-            self._viewer.get_layers_nonthreshold()[i].name
-            for i in self._image_list.get_checked_rows()
-        ]
+        existing_selection: List[str] = []
+        for i in self._image_list.get_checked_rows():
+            item: Optional[QListWidgetItem] = self._image_list.item(i)
+            if item is not None:
```

Comment:
What is `item` ?

Reply:
but what is the value of it in this context?  Item as a var name tells me nothing.  comment or use a descriptive name?